### DISCUSSION TITLE
Get list of possible parameters programatically?

### DISCUSSION THREAD

**Discussion by che85** [SuperElastix/elastix#841] [Ideas]
Hello, 

I saw the help page https://elastix.lumc.nl/doxygen/parameter.html and I was wondering if there is a way to get all possible parameters for the elastix parameter-file including `parameter name`, `parameter type`, and `maximum number of values`?

**Comment by che85** [comment]
@devlaurence This could also be interesting to you.

**Comment by ntatsisk** [comment]
Hi @che85, as far as I know such functionality is not implemented right now in Elastix. 

@N-Dekker, @mstaring, What are your thoughts on such feature?
